### ISSUE TITLE
Doesn't work with demo version, crashes at startup.

### ISSUE THREAD

**Issue by Slipyx** [tobiaspfaff/grim_mouse#37]
Trying this modification using the demo data files doesn't seem to work as it crashes when trying to start game. I have all the files set right including the update and even the patch from the official residual distribution. Log mentions "WARNING: Could not find bitmap cursor0.tga!" though not too sure if this is the real cause for crash.

**Comment by Slipyx** [comment]
Guess I should mention I'm using latest windows 0.5 version, though I do recall same thing happening with a previous version.

**Comment by tobiaspfaff** [comment]
Yes, the mod only works with the full version of the game. Some of the demo's source code files are different from the full version, so hotpatching the game source code does not work any more.
